docker-compose.yml

```diff
@@ -342,6 +342,8 @@ services:
     depends_on:
       envsubst:
         condition: service_completed_successfully
+    environment:
+      CUSTOM_SCRIPTS_DIRECTORY: /etc/georchestra/datafeeder/ui-scripts
```

Comment:
need to update the directory with https://github.com/georchestra/datadir/pull/374#pullrequestreview-1794738699